### PR TITLE
feat(blog): add 1.2 announcement

### PR DESCRIPTION
**Pull request by lucasfernog**

### CODE REVIEW

File: blog/2022-11-08-tauri-1-2.mdx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+# Announcing Tauri 1.2.0
+
+![Tauri 1.2 Launch Hero Image](./tauri_1_2_images/header.png)
+
+The Tauri team is happy to announce the 1.2.0 release. It includes a security fix, so we encourage new and existing users to update to one of the fixed version. Other changes were internally audited and no security issues were found.
```
Comment (Beanow) [inline blog/2022-11-08-tauri-1-2.mdx]: version > versions

**Review by FabianLars — APPROVED** [review]
(Just going to approve it already in case we want to add the hero banner later)

**Comment by Beanow** [comment]
> (Just going to approve it already in case we want to add the hero banner later)

You mean that banner that's already included? :D
image

**Comment by FabianLars** [comment]
The small one on the homepage https://tauri.app/img/index/tauri_1_1_dark.svg

**Pull request merged**